### DISCUSSION TITLE
Tone-Mapping is not working

### DISCUSSION THREAD

**Discussion by base-basebros-online** [google/model-viewer#4505] [Q&A]
The new feature tone-mapping added to model-viewer editor is not working. When you enter the demo link below, you will see that the page initially loads with tone-mapping (which is poster generated by model-viewer editor), but then reverts to its original state.

model-viewer 3.2.0

https://curious-tapioca-34b215.netlify.app/

Thanks,

**Comment by elalish** [comment]
v3.2 is from months ago - v3.3 (with this new feature) should be out this week.

**Reply by base-main-online** [reply]
@elalish yesterday we got mail about [google/model-viewer] Release v3.3.0 - v3.3.0
   
Im trying to use 3.3.0 but it doest not work completely. Any problem with this script?  

<script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.3.0/model-viewer.min.js"></script>
Failed to load resource: the server responded with a status of 404 ()

Live link 
https://resilient-puppy-faebfc.netlify.app/

Thanks,

**Reply by elalish** [reply]
That would be because it isn't up on googleapis yet - working on it. Meanwhile, get it from npm or jsdelivr, etc.
